Dockerfile

```diff
@@ -12,10 +12,17 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -q \
 
 ENV HOME /home/$USER
 
-WORKDIR $HOME/source
+WORKDIR $HOME
 
-COPY osmaxx-py $HOME/source
+# Little trick to avoid rebuilding this image over and over again.
```

Comment:
Instead of stating that this is a 'trick' (whatever that means), this comment should explain or highlight what the trick is and how it works.